jolly-jellyfish/src/page_maker/templates/page_maker/template_create.html

```diff
@@ -0,0 +1,13 @@
+{% extends "page_maker/base.html" %}
+
+{% block content %}
+<div>
```

Comment:
```suggestion
{% block content %}
<h2>Create a Template</h2>
<div>
```
Style finds first element and calls it a header.